### DISCUSSION TITLE
Welcome to Code Hike Discussions!

### DISCUSSION THREAD

**Discussion by pomber** [code-hike/codehike#83] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aiyush-G** [comment]
This project is awesome, I've been looking over the progress over the last few months and am excited to see what else you add! Do you have a roadmap of what features you hope to add and by which dates?

**Reply by Aiyush-G** [reply]
#84 

A roadmap 'discussion' has been made here by @pomber

**Comment by ouiliame** [comment]
This is such an awesome project, and really hard to find somebody who's thinking "reinventing the codeblock" components and about the future of code walkthroughs & etc. I'm wondering -- since there's no existing documentation, I'd be interested in contributing to that effort.

**Reply by pomber** [reply]
Thanks! I'll let you know when I start with the docs.
